<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>CHOOSE OS(EXCLUSIVE)</string>
   </property>
   <widget class="QCheckBox" name="WindowscheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="MaccheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LinuxcheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>NON-EXCLUSIVE</string>
   </property>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Python</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Html</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Java</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>RADIO BUTTON</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GRAB DATA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SET DATA</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
